accelerators/frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/hooks/useCustomMdmsv2.js

```diff
@@ -0,0 +1,26 @@
+import { useQuery, useQueryClient } from "react-query";
+import { Mdmsv2Service } from "../services/elements/Mdmsv2Service";
```

Comment:
@nipunarora-eGov  use custom api hook, also use once workbench folder to create any new hook or util
take a pull and check the hooks that were created inside

Reply:
Updated @jagankumar-egov